src/types.ts

```diff
@@ -2,8 +2,19 @@
 
 import type HandlerFlags from './flags/handler.js';
 
+export interface MediaOptions {
+  quality?: 'best' | '1080' | '720' | '480';
```

Comment:
```suggestion
  videoQuality?: 'best' | '1080' | '720' | '480';
```
needs to be changed everywhere else too